README.md

```diff
@@ -1 +1,135 @@
-# jx-app-jacoco
\ No newline at end of file
+# jx-app-jacoco
+
+jx-app-jacoco provides a means for transferring a Jacoco XML code coverage report 
```

Comment:
For this short README it might not matter, but I like the idea of one line per sentence when using Markdown or AsciiDoc. It makes reviewing changes easier in the long run.

Reply:
Agreed, will update.